truffle/src/main/java/org/jruby/truffle/RubyLanguage.java

```diff
@@ -49,7 +49,8 @@
     public static final int    RUBY_REVISION = 54768;
     public static final String COMPILE_DATE = "2016-12-03";
     public static final String VERSION = "9.1.7.0-SNAPSHOT";
-    public static final String ENGINE = "jruby+truffle";
+    // TODO (pitr-ch 11-Jan-2017): use our value when we are recognized by rubygems
+    public static final String ENGINE = "ruby";
```

Comment:
If this is a one-off change for Bundler, that's what the `ConstantReplacer` class is intended for. It's a hack, for sure, but it's where I've been collecting the gem-specific ones like this.